ai2_kit/domain/dplr.py

```diff
@@ -311,7 +341,6 @@ def dplr_v2_to_v3(data_path: str, sel_symbol: list):
             sel_ids = np.where(np.isin(symbols, sel_symbol))[0]
             n_atoms = len(atype)
 
-            # FIXME: n_frame is undefined
             raw_data = np.load(fname).reshape([n_frames, len(sel_ids), -1])
```

Comment:
my vscode report that n_frames is undefined.

Reply:
Fixed